src/components/Filters/BottomBar/BottomBar.tsx

```diff
@@ -56,12 +64,23 @@ const BottomBar: React.FC<BottomBarProps> = ({
         </Button>
         <Button
           color="primary"
-          disabled={isApplyDisabled}
-          /* disabled={isLoading} TODO enable by https://zitenote.atlassian.net/browse/FEP-1645 */
+          disabled={isApplyDisabled || canCancel}
           variant="solid"
           onClick={onSubmit}
         >
-          Apply
+          {canCancel ? (
```

Comment:
BTW we have loading button state https://securityscorecard.github.io/design-system/alpha/?path=/docs/components-button--playground#loading-buttons

Reply:
Ticket for the update https://zitenote.atlassian.net/browse/FEP-1645 Perhaps is time to tackle it

Reply:
Good point! I've discovered that in the beginning and then forgot about it due to the discrepancy, so thanks for pointing it out :) Is the current loading state used anywhere besides here? if not, I would create a ticket that covers the migration to the new designs for the general Button component.

Reply:
never mind @PrincAm beat me to it 🤣 yeah I'll take care of that!